05/DB.test.js

```diff
@@ -4,9 +4,12 @@ let DBClass;
 
 beforeEach(() => {
 	DBClass = new DB();
+	//a tutaj jakbym wrzucil, a gdzie byloby wiecej to bym nadpisywal w kodzie?
+	// expect.assertions(1);
```

Comment:
Lepiej to zrobić w każdym teście, aby było czytelniej i wiadomo ilu porównań się spodziewamy, ale... można użyć czegoś takiego:

```
beforeEach(() => {
    expect.hasAssertions();
});
```

Wtedy osiągamy to co chcesz :)